### PR TITLE
Blog post covering the future directions for R3

### PR DESCRIPTION
**Pull request by fkr**
This blog post is intended to cover the future directions - post R2.

### CODE REVIEW

**Review by itrich — CHANGES_REQUESTED** [review]
Can you please add a symlink in the German subdirectory? Thank you!

**Comment by fkr** [comment]
> Can you please add a symlink in the German subdirectory? Thank you!

doh! 🤦 
thanks! :)

File: _i18n/en/_posts/blog/2022-03-23-r2-and-future-directions.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title:  "Future directions - the road towards R3"
+category: foo
```
Comment (itrich) [inline _i18n/en/_posts/blog/2022-03-23-r2-and-future-directions.md:4]: Shouldn't we tag this blog post with a more reasonable category? I doubt that we have many RSS subscribers, but who knows :man_shrugging:

Comment (fkr) [reply _i18n/en/_posts/blog/2022-03-23-r2-and-future-directions.md:4]: hahaha. yes, i added that because i did not have the right category at hand :)

Comment (garloff) [reply _i18n/en/_posts/blog/2022-03-23-r2-and-future-directions.md:4]: grep foo on code/docs always gets some funny results.

Comment (fkr) [reply _i18n/en/_posts/blog/2022-03-23-r2-and-future-directions.md]: fixed in a40bc1f1

**Review by itrich — APPROVED** [review]
Looks good now, thank you!

**Pull request merged**